src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -89,7 +115,7 @@ public static boolean isRemoteStorageUrl(String path) {
      * @return an absolute file path if the original path was a relative file path, otherwise the original path
      */
     public static String makeFilePathAbsolute(String path){
-        if (isCloudStorageUrl(path) || isHadoopUrl(path) || isFileUrl(path)){
+        if (isGcsUrl(path) || isHadoopUrl(path) || isFileUrl(path) || isHttpUrl(path)){
             return path;
```

Comment:
It seems like this condition should actually be: "if this is any kind of URL at all, return it unmodified"

Reply:
Yeah, this method is bad.  I guess a check for having a scheme of some sort would be reasonable.  We could also convert to `Path` and call `toAbsolutePath().toUri().toString()` Maybe we should do that.

Reply:
If you attempt the latter, you should be aware that the client code might be relying on paths to local files *not* turning into `file://` URIs...

Reply:
Nope, that doesn't work, it always adds a scheme.

Reply:
Is "contains("://") a valid test for scheme having? Or is that allowed later in a url?  I'm keeping this as is because it's spooky unless we come up with a safe way to do it.

Reply:
You could do `URI uri = new GATKPathSpecifier(path).getURI()`. That will absolutize the path against the local file system only if its not already any kind of URL. Then if that URI has a `file` scheme, return uri.getPath(), which will be the absolute local path.

OTOH I'm hoping to remove this method completely in my next so you could just leave it as is.

Reply:
I added a test for this method as well.

Reply:
@cmnbroad I've left it as is.  Please kill it with fire :)